vpr/src/place/place.cpp

```diff
@@ -485,7 +485,8 @@ void try_place(const t_placer_opts& placer_opts,
 
     std::shared_ptr<SetupTimingInfo> timing_info;
     std::shared_ptr<PlacementDelayCalculator> placement_delay_calc;
-    std::unique_ptr<PlaceDelayModel> place_delay_model;
+    std::unique_ptr<PlaceDelayModel> place_delay_model_holder;
+    const PlaceDelayModel* place_delay_model = nullptr;
```

Comment:
Rather than having two variables in scope which refer to the same object, wouldn't it be clearer to have:
```
std::unique_ptr<PlaceDelayModel> place_delay_model;
```
as before, but just pass `place_delay_mode.get()` to all the subroutine calls?

Reply:
Sure